projects/plugins/jetpack/_inc/client/at-a-glance/stats/dash-stats-bottom.jsx

```diff
@@ -119,6 +120,7 @@ class DashStatsBottom extends Component {
 							/>
 						) }
 						{ this.props.isLinked &&
+							! this.props.isOdysseyStatsEnabled && // Only show if Odyssey Stats is disabled
```

Comment:
I wonder if we could reuse logic like in here?

https://github.com/Automattic/jetpack/blob/12231ae47d994c5b0f9fdf170b38a5478eb26906/projects/plugins/jetpack/_inc/client/traffic/site-stats.jsx#L156

Reply:
Yes, but at the cost of needing to compose the component with withModuleSettingsFormHelpers to gain access to [getOptionValue](https://github.com/Automattic/jetpack/blob/9e2c41cad2b0af9a8629eb8f895bac62b3ac1cae/projects/plugins/jetpack/_inc/client/components/module-settings/with-module-settings-form-helpers.jsx#L140). It might be worth looking into in the future, but likely not high in priority 🙂